src/init.js

```diff
@@ -46,3 +47,8 @@ window.OCA.Talk.registerMessageAction = ({ label, callback, icon }) => {
 	}
 	store.dispatch('addMessageAction', messageAction)
 }
+
+EventBus.$on('Signaling::joinRoom', (payload) => {
+	const token = payload[0]
+	store.dispatch('handleRoomJoined', token)
```

Comment:
I'm a bit worried that we have too many different "joined" states and the naming is going to become confusing.
Here I guess we are "fully joined" which is both the API based joined (`/chat/{token}/active`) and joined on signaling side.

I don't have any better idea currently, maybe we need to work on these confusions separately.

Reply:
We need these 2 joined states in order to speed up the UI... One is "UI joined" and the other one is real, "signaling joined"

Reply:
Before there was only 1 state, so that the UI was waiting for the signaling requests to be done

Reply:
It was not about the state, but about the names.

Reply:
My bad, didn't get this right